frontend/src/Calendar/Legend/LegendItem.css

```diff
@@ -5,6 +5,7 @@
   width: 150px;
   border-left-width: 4px;
   border-left-style: solid;
+  color: var(--white);
```

Comment:
This always gonna be white on every theme?

Reply:
Yes as the backgrounds are dark
![image](https://user-images.githubusercontent.com/1936903/170807086-ff6a8997-97c3-4942-8540-67864a39bfed.png)
![image](https://user-images.githubusercontent.com/1936903/170807095-1a891abd-eb7a-4a85-8491-097a4ada6421.png)

Reply:
Downloaded and Missing should not have white indicators there